tests/test_users.py

```diff
@@ -19,7 +19,7 @@ def test_basic_commands(self, test_db):
         # Delete the same user from the database
         deleted_user = User.delete(test_user.username)
 
-        assert test_user.id == 1
+        assert test_user.id == 4
```

Comment:
I know that we have 3 users when setting up the database, but we should not rely on that for an assertion here. You could check that the user ID  should equal to the number of users in the database.